### ISSUE TITLE
Define Slack Channels

### ISSUE THREAD

**Issue by shopify-services** [Shopify/rbi#1]
# Define Slack Channels

## TL;DR
Associating a team's Slack channel(s) to their services and components makes it easier to find the right people to answer questions about those services. Please add your team's Slack channel(s) to your `service.yml` file, old/archived channel names must be updated.

Define your team's Slack channel(s) in your `service.yml` file as follows:
``` yml
slack_channels:
- your-slack-channel
```

## Why is this being asked?
The first value is also used by spy and other services to announce maintenance periods or incidents to service and component owners.

## What will happen if it doesn't get done within the expected timeframe?
This is mandatory. If this is not done you won't be able to promote this service to the next tier, and it might be reverted to Tier 4 otherwise.

## This doesn't apply to my service. What do I do?
First, leave a comment explaining why it doesn't apply. Then, leave another comment as `/not_applicable`, and close the issue.
If you change your mind, make sure to comment any reason and reopen the issue. Once you've fixed it, services-db will close the issue for you next day.

## I have questions/concerns about this
Please contact the Production Excellence team using Slack at #production-excellence-team.

**Your service**: [rbi/production](https://services.shopify.io/services/rbi/production)
